haystack/nodes/retriever/_openai_encoder.py

```diff
@@ -22,7 +24,17 @@
 class _OpenAIEmbeddingEncoder(_BaseEmbeddingEncoder):
     def __init__(self, retriever: "EmbeddingRetriever"):
         # See https://beta.openai.com/docs/guides/embeddings for more details
```

Comment:
Since we are here, could we update this URL to `https://platform.openai.com/docs/guides/embeddings`?

Reply:
Great point @sjrl. I'll also add the Azure OpenAI link, which has that input diff. (array x string).